src/shared/components/dateRangePicker/utils.ts

```diff
@@ -0,0 +1,13 @@
+import {isISODate} from 'src/shared/utils/dateTimeUtils'
+
+import {isValidStrictly} from 'src/utils/datetime/validator'
+
+export const isValidDatepickerFormat = (d: string): boolean => {
```

Comment:
i wish there was a way to use the wasm to validate these things as the validation is defined in there and is already on the page